core/ui/legacy/theme2/common/src/main/res/values/themes.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <!-- Base main themes that can be easily replaced -->
+    <style name="Theme2.Main.Light.Base" parent="Theme.Material3.Light.NoActionBar" />
+    <style name="Theme2.Main.Dark.Base" parent="Theme.Material3.Dark.NoActionBar" />
+    <style name="Theme2.Main.DayNight.Base" parent="Theme.Material3.DayNight.NoActionBar" />
+
+    <!-- Common main themes -->
+    <style name="Theme2.Main.Light.Common" parent="Theme2.Main.Light.Base">
+        <!-- Draw action mode above content -->
+        <item name="windowActionModeOverlay">true</item>
+    </style>
+
+    <style name="Theme2.Main.Dark.Common" parent="Theme2.Main.Dark.Base">
+        <!-- Draw action mode above content -->
+        <item name="windowActionModeOverlay">true</item>
+    </style>
+
+    <style name="Theme2.Main.DayNight.Common" parent="Theme2.Main.Light.Base" />
+
+    <!-- Main themes -->
+    <style name="Theme2.Main.Light" parent="Theme2.Main.Light.Common">
+        <item name="android:statusBarColor">?attr/colorSurfaceInverse</item>
+        <item name="android:navigationBarColor">?attr/colorSurfaceInverse</item>
```

Comment:
Possible future improvement: These are Android version specific configurations. Maybe they should go into `values-v21/themes.xml`. Same with the contents of `Theme2.Main.Dark` and `Theme2.Main.Startup`.

Reply:
I'll give it a try with the next iteration.